os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/common/src/cycle_queue.c

```diff
@@ -93,18 +94,20 @@ bool CycQueueWrite(uint8_t *pWriteBuf, uint16_t length)
     bool     ret = true;
     uint32_t s;
 
+    s = os_lock();
+
     if (cyc_buffer == NULL)
     {
+        //dbg("cyc_buffer is init\r\n");

```

Comment:
Could you please remove unused code if it is not necessary?

Reply:
Done